http/src/request/channel/invite/create_invite.rs

```diff
@@ -218,9 +218,36 @@ impl<'a> CreateInvite<'a> {
         self.target_user_id(target_user_id)
     }
 
+    /// Set the target type for this invite.
+    pub fn target_type(mut self, target_type: TargetType) -> Self {
+        self.fields.target_type.replace(target_type);
+
+        self
+    }
+
     /// Set the target user type for this invite.
-    pub fn target_user_type(mut self, target_user_type: TargetUserType) -> Self {
-        self.fields.target_user_type.replace(target_user_type);
+    #[deprecated(since = "0.4.1", note = "Use `target_type` instead")]
+    pub fn target_user_type(self, target_user_type: TargetType) -> Self {
```

Comment:
In order to keep this from breaking, how about we create an export of `TargetType` as `TargetUserType` and then deprecate it, like:

```rust
#[deprecated(since = "0.4.1", note = "renamed to `TargetType`")]
pub type TargetUserType = TargetType;